CustomApps/lyrics-plus/index.js

```diff
@@ -328,6 +335,18 @@ class LyricsContainer extends react.Component {
 				lyricContainerUpdate && lyricContainerUpdate();
 			});
 		});
+
+		[
+			["hangul", "hangul"],
+			["romaja", "romaja"]
+		].map(params => {
```

Comment:
Created array has no value and is not needed.
```suggestion
		].forEach(params => {
```